tests/macros/test-toolssidebar.js

```diff
@@ -9,19 +9,19 @@ const utils = require('./utils'),
 
 const locales = {
   'en-US': {
-    'Page_Inspector': 'Page Inspector'
+    'Core_Tools': 'Core Tools'
   },
   'fr': {
-    'Page_Inspector': 'Inspecteur'
+    'Core_Tools': 'Outils principaux'
   }
 };
 
 function checkSidebarDom(dom, locale) {
   let section = dom.querySelector('section');
   assert(section.classList.contains('Quick_links'), 'Section does not contain Quick_links class');
 
-  let links = dom.querySelectorAll('a');
-  assert.equal(links[1].textContent,  locales[locale].Page_Inspector);
+  let summaries = dom.querySelectorAll('summary');
+  assert.equal(summaries[0].textContent,  locales[locale].Core_Tools);
```

Comment:
Weird test, but works after change. The HTML was:

```
<ol>
    <li><a href="#">Core Tools</a>
      <ol>
        <li><a href="/en-US/docs/Tools/Page_Inspector">Page Inspector</a></li>
```

and now is:

```
<ol>
    <li class="toggle">
      <details>
        <summary>Core Tools</summary>
        <ol>
           <li><a href="/en-US/docs/Tools/Page_Inspector">Page Inspector</a></li>
```

I think the weirdness around testing sidebars is a symptom of a need for re-implementing them. Future work.